### ISSUE TITLE
Does it work for windows?

### ISSUE THREAD

**Issue by VinayN03** [ocropus/hocr-tools#167]

**Comment by stweil** [comment]
Yes, it works in Windows WSL, maybe also with Python 3 for Windows, but we cannot test that ourselves. Maybe others already tried it and can report their results here.
